scripts/monitoring/cron-send-create-app.py

```diff
@@ -213,7 +222,7 @@ def test(config):
     noPodCount = 0
     http_code = 0
 
-    for _ in range(testLoopCountMax):
+    for _ in range(self.args.loopcount):
```

Comment:
optional: after seeing your change I'm thinking this get passed into test() as a default parameter, along with all the other parameters
```
def test(loopCountMax=180, loopDelay=5)
``` 

this way all args (self.args.*) can stay contained to within main()